texttospeech/cloud-client/src/test/java/com/example/texttospeech/SynthesizeTextIT.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.texttospeech;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import java.io.File;
+
+import org.junit.After;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/**
+ * Tests for SynthesizeText sample.
+ */
+@RunWith(JUnit4.class)
+@SuppressWarnings("checkstyle:abbreviationaswordinname")
+public class SynthesizeTextIT {
+
+  private static String OUTPUT = "output.mp3";
+  private static String TEXT = "Hello there.";
+  private static String SSML = "<?xml version=\"1.0\"?>\n"
```

Comment:
It's worth testing to make sure that this works fine with a simple `<speak>Hi</speak>`

(per product)